packages/core/src/components/chip/chip-vars.scss

```
@@ -15,6 +18,9 @@
   --tds-chips-color: var(--tds-white);
   --tds-chips-color-active: var(--tds-white);
   --tds-chips-background: var(--tds-grey-800);
+  --tds-chips-background-disabled: var(--tds-grey-200);
```

```suggestion
  --tds-chips-background-disabled: var(--tds-grey-800);
```
Seems like 800 is one for dark mode. Please double check.

---

Good catch!